code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -241,3 +241,56 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/medical/poultice(location)
+
+/datum/chemical_reaction/cryosenium
+	results = list(/datum/reagent/medicine/cryosenium = 3)
+	required_reagents = list(/datum/reagent/medicine/cryoxadone = 1, /datum/reagent/consumable/ice = 1, /datum/reagent/cryostylane = 1) //consider sterilizine if needed.
+	required_temp = -200
+	optimal_temp = 300
+	overheat_temp = NO_OVERHEAT //There is an overheat - 50 see reaction_step()
+	optimal_ph_min = 4
+	optimal_ph_max = 10
+	determin_ph_range = 6
+	temp_exponent_factor = 0.5
+	ph_exponent_factor = 1
+	thermic_constant = -30
+	H_ion_release = 0
+	rate_up_lim = 25
+	purity_min = 0.15
+	reaction_flags = REACTION_HEAT_ARBITARY
+
+//purity != temp (above 50) - the colder you are the more impure it becomes
+/datum/chemical_reaction/cryosenium/reaction_step(datum/equilibrium/reaction, datum/reagents/holder, delta_t, delta_ph, step_reaction_vol)
+	. = ..()
+	if(holder.chem_temp < 50)
+		overheated(holder, reaction, step_reaction_vol)
+		return
+	var/datum/reagent/medicine/cryosenium/cryo = holder.has_reagent(/datum/reagent/medicine/cryosenium)
```

Comment:
Name this cryosenium here and everywhere else, it'll make it more obvious and not conflict with anything else cryo.

Reply:
Done